pkgs/by-name/in/invoiceplane/package.nix

```diff
@@ -24,23 +25,25 @@ php.buildComposerProject2 (finalAttrs: {
   pname = "invoiceplane";
   inherit version;
 
-  src = fetchFromGitHub {
-    owner = "InvoicePlane";
-    repo = "InvoicePlane";
-    tag = "v${version}";
-    hash = "sha256-XNjdFWP5AEulbPZcMDXYSdDhaLWlgu3nnCSFnjUjGpk=";
+  src = applyPatches {
+    src = fetchFromGitHub {
+      owner = "InvoicePlane";
+      repo = "InvoicePlane";
+      tag = "v${version}";
+      hash = "sha256-XNjdFWP5AEulbPZcMDXYSdDhaLWlgu3nnCSFnjUjGpk=";
+    };
+    patches = fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/InvoicePlane/InvoicePlane/pull/1306.patch";
```

Comment:
The PR is not merged, we should fetch the individual commit instead.

Reply:
In that case, let's use the existing patch file by onny.